ambry-api/src/main/java/com.github.ambry/store/MessageInfo.java

```diff
@@ -64,6 +64,16 @@ public boolean isExpired() {
     return getExpirationTimeInMs() != Utils.Infinite_Time && System.currentTimeMillis() > getExpirationTimeInMs();
   }
 
+  public String getServiceId() {
```

Comment:
At the moment, IndexValue do not have fields for containerId and serviceId yet. This is a temporary solution for obtaining a mock containerId and serviceId for testing purpose.